src/components/mfa-reset-with-ipv/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiResponseResult, DefaultApiResponse } from "../../types";
+import { Request } from "express";
+
+export interface MfaResetAuthorizeResponse extends DefaultApiResponse {
+  location: string;
```

Comment:
Hey just a small thing, the backend currently returns a field called [`authorize_url` not `location` in the response body](https://github.com/govuk-one-login/authentication-api/blob/1e26950a99fffa8da505e4c1ff848c71e9f8df1b/frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/MfaResetResponse.java#L6), feel free to change the backend if it fits better with what the team usually do

Reply:
Thanks Ryan :)